sass/layout/_navbar.scss

```diff
@@ -1,106 +1,110 @@
-  right: 0px;
-  top: 0px;
-}
-
-.nav > .nav-btn > label {
-  display: inline-block;
-  width: 50px;
+.nav {
+  z-index: 100;
   height: 50px;
-  padding: 13px;
-}
-
-.nav > .nav-btn > label > span {
-  display: block;
-  width: 25px;
-  height: 10px;
-  border-top: 2px solid #eee;
-}
-.nav > .nav-links {
-  position: absolute;
-  display: block;
   width: 100%;
   background-color: $color-primary;
-  height: 0px;
-  transition: all 0.3s ease-in;
-  overflow-y: hidden;
-  top: 50px;
-  left: 0px;
-}
-.nav > .nav-links > a {
-  display: block;
-  width: 100%;
-}
-.nav > .nav-links > input {
-  display: block;
-  width: 80%;
-  margin: 1rem;
-}
-.nav > #nav-check:not(:checked) ~ .nav-links {
-  height: 0px;
-}
-.nav > #nav-check:checked ~ .nav-links {
-  height: calc(100vh - 50px);
-  overflow-y: auto;
-}
-
-.nav > .nav-links {
-  display: inline;
-  float: right;
-  font-size: 18px;
-}
-
-.nav > .nav-links > a {
-  display: inline-block;
-  padding: 13px 10px 13px 10px;
-  text-decoration: none;
-  color: #efefef;
-}
-
-@include respond(phone) {
-  .nav > .nav-btn > label:hover,
-  .nav #nav-check:checked ~ .nav-btn > label {
-    background-color: rgba(0, 0, 0, 0.3);
-  }
+  position: relative;
+  justify-content: space-between;
 
-  .nav > .nav-header {
+  &-header {
     display: inline;
+    line-height: 3rem;
   }
 
-  .nav > .nav-header > .nav-title {
+  &-title {
     display: inline-block;
     font-size: 22px;
     color: #fff;
-    padding: 10px 10px 10px 10px;
+    padding: 10px;
   }
 
-  .nav > .nav-btn {
+  &-btn {
     display: none;
+
+    @media (max-width: 600px) {
+      display: inline-block;
+      position: absolute;
+      right: 0px;
+      top: 0px;
+    }
+
+    & label {
+      @media (max-width: 600px) {
+        display: inline-block;
+        width: 50px;
+        height: 50px;
+        padding: 13px;
+      }
+
+      & span {
+        @media (max-width: 600px) {
+          display: block;
+          width: 25px;
+          height: 10px;
+          border-top: 2px solid #eee;
+        }
+      }
+    }
   }
 
-  .nav > .nav-links > a:hover {
-    background-color: rgba(0, 0, 0, 0.3);
+  &-links {
+    display: inline;
+    float: right;
+    font-size: 1.6rem;
+
+    & a {
+      display: inline-block;
+      padding: 13px 10px 13px 10px;
+      text-decoration: none;
+      color: #efefef;
+
+      @media (max-width: 600px) {
+        display: block;
+        width: 100%;
+      }
+
+      &:hover {
+        background-color: rgba(0, 0, 0, 0.3);
+      }
+    }
+
+    @media (max-width: 600px) {
+      position: absolute;
+      display: block;
+      width: 100%;
+      background-color: $color-primary;
+      height: 0px;
+      transition: all 0.3s ease-in;
+      overflow-y: hidden;
+      top: 50px;
+      left: 0px;
+    }
+  }
+
+  #nav-check {
+    display: none;
   }
 }
-.nav > #nav-check {
-  display: none;
+
+@media (max-width: 600px) {
+
+  .nav > #nav-check:not(:checked) ~ .nav-links {
```

Comment:
Since id selectors are unique. There's little need of having `.nav > ` in front ;)